src/test/controller/EnumController.test.ts

```diff
@@ -11,11 +15,14 @@ import { RedisClient } from 'redis-mock';
 describe('Unit tests for PromotionController', function () {
   let app: Express;
   let redisClient: RedisClient;
+  let admin: any;
```

Comment:
Is there a more specific type here?

Reply:
I haven't found the good type expression for firebase mock auth. When I set `Auth` defined in `firebase admin sdk`, it does not fit.